jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -47,6 +48,21 @@ function configure_mem_ratio() {
     export JAVA_INITIAL_MEM_RATIO=${JAVA_INITIAL_MEM_RATIO:-25}
 }
 
+function configure_logger_config_file() {
+  # JUL implementation: https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html
+  local allowed_log_levels=("ALL" "CONFIG" "FINE" "FINER" "FINEST" "INFO" "OFF" "SEVERE" "WARNING")
+
+  if [[ ! "${allowed_log_levels[@]}" =~ "${LOG_LEVEL}" ]]; then
+    log_warning "Log Level ${LOG_LEVEL} is not allowed, the allowed levels are ${allowed_log_levels[@]}"
+  else
+	  local log_level=${LOG_LEVEL:-INFO}
+    sed -i "s/{LOG_LEVEL}/${log_level}/" ${CONFIG_DIR}/logging.properties
+    local logger_categories=$(echo $LOGGER_CATEGORIES | awk '{gsub(",", "\\n")}1')
+    sed -i "s/{PACKAGES_LOG_LEVEL}/${logger_categories}/" ${CONFIG_DIR}/logging.properties
+    JAVA_OPTS_APPEND="${JAVA_OPTS_APPEND} -Djava.util.logging.config.file=${CONFIG_DIR}/logging.properties"
```

Comment:
add a log_info here to print what was configured.

Reply:
ok